lms/templates/widgets/footer-language-selector.html

```diff
@@ -18,20 +18,26 @@
 <%namespace name='static' file='../static_content.html'/>
 
 <div class="footer-language-selector">
-    <label for="footer-language-select">
-        <span class="icon fa fa-globe" aria-hidden="true"></span>
-        <span class="sr">${_("Choose Language")}</span>
-    </label>
-    <select id="footer-language-select" name="language" onchange="footerLanguageSelector.handleSelection(this)">
-        % for language in sorted(released_languages(), key=lambda x: x.code):
-            <% language_name = Locale.parse(language.code.replace('_', '-'), sep='-').language_name %>
-            % if language.code == LANGUAGE_CODE:
-                <option value="${language.code}" selected="selected">${language_name}</option>
-            % else:
-                <option value="${language.code}">${language_name}</option>
-            % endif
-        % endfor
-    </select>
+    <div role="group" aria-label="Changing the value results in a page reload.">
```

Comment:
This aria-label doesn't really make sense to me. Can we change it to something like `Change language and reload the page.`

Reply:
sure, I will change that

Reply:
@fysheets  on second thought, changing it to ```Change language and reload the page.``` might give an impression that users need to explicitly reload the page.
The information we want to pass to the users is that by changing the language the page reloads automatically

Reply:
Okay! Confirmed that we don't need to note that the page will reload: `Change page language` should be good here.